hoodie-common/src/main/java/com/uber/hoodie/common/util/HoodieAvroUtils.java

```diff
@@ -124,6 +124,23 @@ public static Schema addMetadataFields(Schema schema) {
     return mergedSchema;
   }
 
+  /**
+   * Adds the Hoodie metadata fields to the given schema
+   */
+  public static Schema dropMetadataFields(Schema schema) {
```

Comment:
Is this method name correct ? dropMetadataFields ?

Reply:
The java-doc is obviously wrong. I will fix it. The method name is correct. We copy the schema and drop the meta-data fields while doing it.

Reply:
Ended up removing this method as it is was not used.